middleware/cmd/middleware/main.go

```diff
@@ -53,7 +53,10 @@ func main() {
 		}
 	}
 	defer logBeforeExit()
-	middleware := middleware.NewMiddleware(argumentMap, false)
+	middleware, err := middleware.NewMiddleware(argumentMap, false)
+	if err != nil {
+		log.Fatal("error starting the middleware, is redis connected? \nIf you are running the middleware outside of the base consider setting the redis mock flag to true.")
```

Comment:
Please print at least the actual `err` error message! I think the error message is way to detailed. The suggestion will be outdated and confusing as soon as the another component initialized in the middlware returns an error.   

@Stadicus send me the picture of this log output below. He asked why the middleware can't connect to Redis, even if it's running. I'm guessing it's due to not having the Redis keys for the authentication setup in Redis set. It would be way easier to debug if it would say that the Redis key can't be found...
 
![image](https://user-images.githubusercontent.com/19157360/67686009-505a0c00-f996-11e9-8e9a-d84d5bfb3eba.png)

Reply:
I think it's an old image and has a newer middleware binary on it. I don't have access to the device. @Stadicus probably knows more.

Reply:
I'm also logging the error now.